File: DataScienseProject/Models/GalleryModel.cs

using System.Collections.Generic;

namespace DataScienseProject.Models
{
    public class GalleryModel
    {
        public long ViewKey { get; set; }
        public string ViewName { get; set; }
        public string Image { get; set; }
        public List<string> Executors { get; set; }
        public List<string> Tags { get; set; }
        public List<string> ShortDescription { get; set; }
        public int OrderNumber { get; set; }
    }
}

File: DataScienseProject/ClientApp/src/app/models/gallery/gallery.model.ts
import { ExecutorModel } from "../executor.model"

export class GalleryModel {
  viewKey: number;
  viewName: string;
  image: string;
  executors: Array<ExecutorModel>;
  tags: Array<string>;
  shortDescription: Array<string>;
  orderNumber?: number;  
}
